website/react-magma-docs/src/pages/design-intro/get-started.mdx

```diff
@@ -26,36 +26,15 @@ We also welcome contributions from people outside the React Magma team. This inc
 
 ## Resources & Tools
 
-<div class="heading-leading-img">
-  <Heading level={3}>
-    <SketchLogo style={{ width: '40px' }} />
-    Sketch
-  </Heading>
-</div>
-
-Sketch is the primary application we use for designing products within Cengage. We maintain a comprehensive set of React Magma Sketch libraries containing type styles, icons, and components. If you are a designer at Cengage, but you are not an Abstract user, please reach out to the Course Solutions UX team if you need access to the libraries.
+At Cengage we use Figma for our primary design tool and we have component libraries available for you to use. To access these libraries you'll need to sign in to the Cengage Figma organization using your Cengage email address.
```

Comment:
At the top of this file we import the svgs:
```
import SketchLogo from '../../images/logos/sketch-symbol.svg';
import InvisionLogo from '../../images/logos/invision_icon_pink.svg';
import AbstractLogo from '../../images/logos/abstract-logo.svg';
```

Do you mind removing the imports and the actual SVGs, since we don't use them anywhere else?

Reply:
Done.